tools/c7n_left/tests/test_left.py

```diff
@@ -49,7 +49,9 @@ def on_results(self, results):
 
 def run_policy(policy, terraform_dir, tmp_path):
     (tmp_path / "policies.json").write_text(json.dumps({"policies": [policy]}, indent=2))
-    config = Config.empty(policy_dir=tmp_path, source_dir=terraform_dir, exec_filter=None)
+    config = Config.empty(
+        policy_dir=tmp_path, source_dir=terraform_dir, exec_filter=None, var_file=""
```

Comment:
Why and empty string here for `var_file` when `None` is used above?

Reply:
tfparse passes through literal none if given one, the other should get changed to an empty string, which would make the handling uniform.. right until we start passing arrays of paths ;-) anyways atm None w get string converted at the c level to None, which didn't match any paths 🍭 , some of that can be cleaned up in tfparse, but this is a simple workaround.